lib/src/Uncertainty/Bayesian/GaussianNonLinearCalibration.cxx

```diff
@@ -389,7 +389,16 @@ Point GaussianNonLinearCalibration::run(const Sample & inputObservations,
   LeastSquaresProblem problem(residualFunction);
   algorithm_.setVerbose(true);
   algorithm_.setProblem(problem);
-  algorithm_.setStartingPoint(getCandidate());
+  try
+  {
+    // If the solver is single start, we can use its setStartingPoint method
+    algorithm_.setStartingPoint(getCandidate());
+  }
+  catch (NotDefinedException &) // setStartingPoint is not defined for the solver
+  {
+    LOGWARN(OSS() << "Candidate=" << getCandidate() << " is ignored because algorithm_ "
```

Comment:
you can remove the underscore because now algorithm refers to the name that follows and no more to the class member.